arch/cpu/simplelink-cc13xx-cc26xx/rf-settings/cc13x0/prop-settings.c

```diff
@@ -66,13 +66,27 @@
 #include "prop-settings.h"
 /*---------------------------------------------------------------------------*/
 /* TI-RTOS RF Mode Object */
+#if defined(DEVICE_CC1310)
```

Comment:
```
#if defined(DEVICE_CC1310)
#define RF_MODE RF_MODE_PROPRIETARY_SUB_1
#else
#define RF_MODE RF_MODE_MULTIPLE
#endif
```

and then we can have a single definition for the `rf_prop_mode` struct?

Reply:
I will call it RF_PROP_MODE, as RF_MODE is already used by the generic RF configuration.

Reply:
Any name for the macro, really. I just pulled this one out of my head without checking elsewhere. I don't feel strongly, just a thought